christfellowshipchurch/web-app-v2
# Add community sidebar search

This PR adds the filtering sidebar to the community search page on large screens. 

TO TEST:
Click around make sure everything works as expected. No functionality should have changed from the modal version on smaller screens.

Closes #143 

![image](https://user-images.githubusercontent.com/2528817/117217759-8831e300-adc7-11eb-829a-6e9f047a0e05.png)
